src/turnkeyml/cli/report.py

```diff
@@ -78,11 +78,34 @@ def summary_spreadsheets(args) -> None:
 
                         # Copy the build-specific stats
                         for key, value in build.items():
-                            # Break each value in "completed build stages" into its own column
-                            # to make analysis easier
+                            # Break each value in "completed build stages" into status and duration
+                            # to make the analysis of this data easier
                             if key == fs.Keys.COMPLETED_BUILD_STAGES:
-                                for subkey, subvalue in value.items():
-                                    evaluation_stats[subkey] = subvalue
+                                previous_state_incomplete = False
+                                for stage in build[fs.Keys.SELECTED_SEQUENCE_OF_STAGES]:
+                                    duration_column_name = f"stage_duration: {stage}"
+                                    state_column_name = f"stage_status: {stage}"
```

Comment:
```suggestion
                                    duration_column_name = f"stage_duration:{stage}"
                                    state_column_name = f"stage_status:{stage}"
```

Please remove the whitespace because it makes the raw data harder to parse.

If you have a string like `stage_duration: export_pytorch` and use `.split(":")` or Excel's data-to-columns feature, you end up with `["stage_duration", " export_pytorch"]`. Note the leading whitespace on ` export_pytorch`.

Reply:
Changed as suggested